harvard-lts/epadd-curator-app
# Update OSN

**Update OSN**
* * *

**Ticket Link**: [LTSEPADD-29](https://jira.huit.harvard.edu/browse/LTSEPADD-29)

# What does this Pull Request do?
This PR allows the use to supply a name that will be used in DRS associated with their export. The tests were also updated so OSN can be tested. If no OSN is supplied during a test, the OSN will be set to test_{timestamp}.

# How should this be tested?

Already deployed to dev
1. Empty epadd-export-dev bucker with `aws s3 rm --recursive s3://epadd-export-dev/test/ --profile epadd-export-dev-rw   `
2. Call `http://ltsds-cloud-dev-1.lib.harvard.edu:10586/testExport/{aUniqueOSN}`
3. Confirm that the batch is ingested with given OSN

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n/a
- integration tests? Yes

# Interested parties
@ives1227
